README.md

```diff
@@ -5,68 +5,45 @@
+ tools, libraries, and its overall programming ecosystem to implement complex systems with ease.
 
-* Define workflows in code. No JSON schemas or designers are needed.
-* Call other functions synchronously and asynchronously. Output from called functions can be saved to local variables.
-* Automatically checkpoint progress whenever the function schedules async work. Local state is never lost if the process recycles or the VM reboots.
+ 🐍  Find us on PyPi [here](https://pypi.org/project/azure-functions-durable/) 🐍   
+
+
+## More than just stateful Azure Functions
+Durable Functions is offered as an extension of [Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/functions-overview). The extension manages state, checkpoints, and restarts for you. Python Durable Functions' advantages include:
+
+* Defining workflows as code. No JSON schemas or designers are needed.
+* Managing return values, exceptions, and control-flow using familiar Python constructs.
+* Scheduling serverless functions synchronously and asynchronously.
+* Automatically checkpointing progress whenever the function schedules async work. Local state is never lost if the process recycles or the VM reboots.
 
 You can find more information at the following links:
 
 * [Azure Functions overview](https://docs.microsoft.com/en-us/azure/azure-functions/functions-overview)
 * [Azure Functions Python developers guide](https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-python)
 * [Durable Functions overview](https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-overview?tabs=python)
 
-A durable function, or _orchestration_, is a solution made up of different types of Azure Functions:
-
-* **Activity:** the functions and tasks being orchestrated by your workflow.
-* **Orchestrator:** a function that describes the way and order actions are executed in code.
-* **Client:** the entry point for creating an instance of a durable orchestration.
+## Core Concepts
 
-Durable Functions' function types and features are documented in-depth [here.](https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-types-features-overview)
+A Durable Functions application is a solution made up of Azure Functions serving one of four distinct roles. These roles are outline below: 
 
-## Current limitations
+* **Activity:** they are the basic unit of work, a task to be scheduled.
+* **Orchestrator:** they describe which actions to execute, in what order to schedule them, deal with exceptions, etc. As their name implies, they organize and manage a workflow to completion.
+* **Entity:**  They allow you to read and update small pieces of state. Unlike orchestrators, entities manage state explicitely instead of implicitely managing it via control flow.
```

Comment:
Not sure this description covers the entirety of what entities are for.

Also nit: "explicitly" and "implicitly" are spelled wrong.

Reply:
I ended up removing this specific section and just redirecting to the Microsoft docs